docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-creating-a-case-proc.adoc

```diff
@@ -1,3 +1,32 @@
 [id='case-management-creating-a-case-proc']
 = Creating a case in {PRODUCT}
 
+A case project is a type of project in {CENTRAL} that is configured for case management. When you create a new case project, the following configurations are set automatically in {CENTRAL}:
+
+* Runtime strategy is set to Per Case
```

Comment:
No period after the 1st bullet. Also, is the term "Per Case" already defined somewhere? May just be me, but I think a simple intro to what that setting really is would help.